Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -619,7 +619,7 @@
     <string name="settings_notifications_illustration_description_inactive">"Жена, която е дезактивирала известията от приложението Corona-Warn-App."</string>
     <!-- XBUT: settings - go to reset application -->
     <string name="settings_reset_title">"Изчистване на данните в приложението"</string>
-    <!-- XTXT: settings(reset) - explains the user what do expect when he navigates to reset -->
+    <!-- XTXT: settings(reset) - explains the user what do expect when she or he navigates to reset -->
```

Comment:
just a short question: Was it recommended to replace it with "she or he"?
It's a little bit strange to read as the user is a "general term"..
Maybe something like: "explains the user what to do when navigating to reset" would also be fine..
But I really don't know, just thinking..

Reply:
a user can be male or female, he would only include men

Reply:
It's strange because you are used to language that only addresses men